charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/statefulset.yaml

```diff
@@ -91,6 +91,30 @@ spec:
         - name: config
           mountPath: /usr/share/elasticsearch/config/jvm.options
           subPath: jvm.options
+      - name: metrics-exporter
+        image: {{ index .Values.global.images "elasticsearch-metrics-exporter" }}
+        image: justwatch/elasticsearch_exporter:1.0.2
+        imagePullPolicy: IfNotPresent
+        command: ["elasticsearch_exporter",
```

Comment:
Any reason for not using just using `args` here? elasticsearch_exporter is the entrypoint and you just need to pass it the arguments.